### ISSUE TITLE
[Feedback] Features - /features/

### ISSUE THREAD

**Issue by BlackDahlia313** [discord-tickets/docs#33]
Comparisons links to a 404. Looks like blog was redone recently :)

**Comment by eartharoid** [comment]
Yes, the blog is quite new and I haven't written the comparisons yet. 😁 Thanks for reminding me.
